mne/gui/_coreg.py

```diff
@@ -1422,6 +1430,7 @@ def _configure_dock(self):
             callback=self._reset_fitting_parameters,
             tooltip="Reset all the fitting parameters to default value",
             layout=fitting_options_layout,
+            style='pushbutton',
```

Comment:
I would not add `style='pushbutton'` to all of these. I think `pushbutton` is a clear and obvious default for that function (and should have always been that way). Adding this param to all of these calls makes it less clear to me actually, in that at first it seems like you *are* somehow deviating from whatever the reasonable defaults are. (When reading through this code I thought, "why isn't the default `'pushbutton'` already?" just to read farther down that it actually is.)

Reply:
Very good point, I will adjust this accordingly.